WordPress/Classes/ViewRelated/Site Creation/FinalAssembly/SiteAssemblyStep.swift

```diff
@@ -25,9 +25,9 @@ final class SiteAssemblyStep: WizardStep {
     /// - Parameters:
     ///   - creator: the in-flight creation instance
     ///   - service: the service to use for initiating site creation
-    init(creator: SiteCreator, service: SiteAssemblyService) {
+    init(creator: SiteCreator, service: SiteAssemblyService, onDismissEpilogue: (() -> Void)? = nil) {
```

Comment:
Same as with the quick start prompt, I'm not sure that this needs to reference the epilogue specifically, as the site assembly flow can be displayed at other times too.

Also, we should add this parameter to the parameters doc comment above.